app/Language/en/Validation.php

```diff
@@ -1,4 +1,12 @@
 <?php
 
 // override core en language system validation or define your own en language validation message
-return [];
+return [
+    'missingField' => 'The {field} field is required.',
+    'missingLowercaseLetter' => 'The {field} field must be at least one lowercase letter.',
+    'missingUppercaseLetter' => 'The {field} field must be at least one uppercase letter.',
+    'missingNumber' => 'The {field} field must have at least one number.',
+    'missingSpecialCharacter' => 'The {field} field must have at least one special character.',
+    'tooShort' => 'The {field} field must be at least 8 characters in length.',
```

Comment:
Is it smart to hardcode the 8 here?
I feel like you will forget to edit it when  you edit the validation.

Reply:
Good catch! I'll introduce a named constant.